papyrus/renderers.py

```diff
@@ -175,6 +183,10 @@ def _render(cls, system):
             if request is not None:
                 response = request.response
                 response.content_type = 'application/xml'
-                io = self.generator.get_class_xsd(StringIO(), cls)
+                if PY == 2:
+                    io = StringIO
+                else:
+                    io = BytesIO
```

Comment:
Can't we do better using `six`? Does `six` have abstractions for StringIO related stuff?